### PR TITLE
Adding unit test draft 

### PR DESCRIPTION
**Pull request by gab-shell**

### CODE REVIEW

File: tests/test_checks.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+# Third party imports
+import pytest
+
+# Custom imports
+from redata.checks.data_delayed import (
+    check_data_delayed
+)
+
+def test_check_data_delayed():
```
Comment (mateuszklimek) [inline tests/test_checks.py:11]: I would probably focus on getting postgres details in this function, with debugging (adding pdb inside)

Comment (gab-shell) [reply tests/test_checks.py:13]: If in this I uncomment 
```
from redata.checks.data_delayed import (
    check_data_delayed
)
```
I get
```
$ pytest -q

=================================================== ERRORS ====================================================
____________________________________ ERROR collecting tests/test_checks.py ____________________________________
venv/lib/python3.8/site-packages/sqlalchemy/engine/base.py:2336: in _wrap_pool_connect
    return fn()
venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py:304: in unique_connection
    return _ConnectionFairy._checkout(self)
venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py:778: in _checkout
    fairy = _ConnectionRecord.checkout(pool)
venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py:495: in checkout
    rec = pool._do_get()
venv/lib/python3.8/site-packages/sqlalchemy/pool/impl.py:140: in _do_get
    self._dec_overflow()
venv/lib/python3.8/site-packages/sqlalchemy/util/langhelpers.py:68: in __exit__
    compat.raise_(
venv/lib/python3.8/site-packages/sqlalchemy/util/compat.py:182: in raise_
    raise exception
venv/lib/python3.8/site-packages/sqlalchemy/pool/impl.py:137: in _do_get
    return self._create_connection()
venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py:309: in _create_connection
    return _ConnectionRecord(self)
venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py:440: in __init__
    self.__connect(first_connect_check=True)
venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py:661: in __connect
    pool.logger.debug("Error on connect(): %s", e)
venv/lib/python3.8/site-packages/sqlalchemy/util/langhelpers.py:68: in __exit__
    compat.raise_(
venv/lib/python3.8/site-packages/sqlalchemy/util/compat.py:182: in raise_
    raise exception
venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py:656: in __connect
    connection = pool._invoke_creator(self)
venv/lib/python3.8/site-packages/sqlalchemy/engine/strategies.py:114: in connect
    return dialect.connect(*cargs, **cparams)
venv/lib/python3.8/site-packages/sqlalchemy/engine/default.py:509: in connect
    return self.dbapi.connect(*cargs, **cparams)
venv/lib/python3.8/site-packages/psycopg2/__init__.py:127: in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
E   psycopg2.OperationalError: FATAL:  password authentication failed for user "admin"
E   FATAL:  password authentication failed for user "admin"

The above exception was the direct cause of the following exception:
tests/test_checks.py:8: in <module>
    from redata.checks.data_delayed import (
redata/checks/data_delayed.py:1: in <module>
    from redata.db_operations import metrics_db, metadata
redata/db_operations.py:40: in <module>
    metadata.reflect(bind=metrics_db)
venv/lib/python3.8/site-packages/sqlalchemy/sql/schema.py:4448: in reflect
    with bind.connect() as conn:
venv/lib/python3.8/site-packages/sqlalchemy/engine/base.py:2263: in connect
    return self._connection_cls(self, **kwargs)
venv/lib/python3.8/site-packages/sqlalchemy/engine/base.py:104: in __init__
    else engine.raw_connection()
venv/lib/python3.8/site-packages/sqlalchemy/engine/base.py:2369: in raw_connection
    return self._wrap_pool_connect(
venv/lib/python3.8/site-packages/sqlalchemy/engine/base.py:2339: in _wrap_pool_connect
    Connection._handle_dbapi_exception_noconnection(
venv/lib/python3.8/site-packages/sqlalchemy/engine/base.py:1583: in _handle_dbapi_exception_noconnection
    util.raise_(
venv/lib/python3.8/site-packages/sqlalchemy/util/compat.py:182: in raise_
    raise exception
venv/lib/python3.8/site-packages/sqlalchemy/engine/base.py:2336: in _wrap_pool_connect
    return fn()
venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py:304: in unique_connection
    return _ConnectionFairy._checkout(self)
venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py:778: in _checkout
    fairy = _ConnectionRecord.checkout(pool)
venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py:495: in checkout
    rec = pool._do_get()
venv/lib/python3.8/site-packages/sqlalchemy/pool/impl.py:140: in _do_get
    self._dec_overflow()
venv/lib/python3.8/site-packages/sqlalchemy/util/langhelpers.py:68: in __exit__
    compat.raise_(
venv/lib/python3.8/site-packages/sqlalchemy/util/compat.py:182: in raise_
    raise exception
venv/lib/python3.8/site-packages/sqlalchemy/pool/impl.py:137: in _do_get
    return self._create_connection()
venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py:309: in _create_connection
    return _ConnectionRecord(self)
venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py:440: in __init__
    self.__connect(first_connect_check=True)
venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py:661: in __connect
    pool.logger.debug("Error on connect(): %s", e)
venv/lib/python3.8/site-packages/sqlalchemy/util/langhelpers.py:68: in __exit__
    compat.raise_(
venv/lib/python3.8/site-packages/sqlalchemy/util/compat.py:182: in raise_
    raise exception
venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py:656: in __connect
    connection = pool._invoke_creator(self)
venv/lib/python3.8/site-packages/sqlalchemy/engine/strategies.py:114: in connect
    return dialect.connect(*cargs, **cparams)
venv/lib/python3.8/site-packages/sqlalchemy/engine/default.py:509: in connect
    return self.dbapi.connect(*cargs, **cparams)
venv/lib/python3.8/site-packages/psycopg2/__init__.py:127: in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
E   sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) FATAL:  password authentication failed for user "admin"
E   FATAL:  password authentication failed for user "admin"
E   
E   (Background on this error at: http://sqlalche.me/e/13/e3q8)
============================================== warnings summary ===============================================
venv/lib/python3.8/site-packages/_pytest/config/__init__.py:1233
  /home/drkg4b/projects/redata/venv/lib/python3.8/site-packages/_pytest/config/__init__.py:1233: PytestConfigWarning: Unknown config option: ostgresql_host
  
    self._warn_or_fail_if_strict(f"Unknown config option: {key}\n")

venv/lib/python3.8/site-packages/sqlalchemy/util/langhelpers.py:263
  /home/drkg4b/projects/redata/venv/lib/python3.8/site-packages/sqlalchemy/util/langhelpers.py:263: SADeprecationWarning: The 'postgres' dialect name has been renamed to 'postgresql'
    loader = self.auto_fn(name)

-- Docs: https://docs.pytest.org/en/stable/warnings.html
=========================================== short test summary info ===========================================
ERROR tests/test_checks.py - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) FATAL:  password a...
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
2 warnings, 1 error in 0.74s
```

File: tests/test_checks.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    cur.execute(INSERT_Q)
+
+    # Get the tables
+    cur.execute(SELECT_Q)
+    # table = cur.fetchall()
+
+    db = Postgres('test', cur)
+    table = MonitoredTable()
+
+    table.setup_for_source_table(db, 'test')
```
Comment (mateuszklimek) [inline tests/test_checks.py:39]: table = MonitoredTable.setup_for_source_table(db, 'test')

I think that's how this line with setup should look like

Comment (gab-shell) [reply tests/test_checks.py:39]: Sure, updated

**Pull request closed without merging**